workspace/events/event.usebuild.php

```diff
@@ -34,31 +34,38 @@ public function load()
 
 	public static function documentation()
 	{
-		return new XMLElement('p', 'This is an event that will look for the ?use-build query string or the use-build cookie. If found, it will set the cookie for this request and output proper data into the xml');
+		return new XMLElement('p', 'This is an event that will look for the ?use-dev query string or the use-dev cookie. If found, it will set the cookie for this request and output proper data into the xml');
 	}
 
 	protected function __trigger()
 	{
 		if (!class_exists('Frontend', false)) {
 			return false;
 		}
-		$isQuery = isset($_GET['use-build']);
-		$cookieName = __SYM_COOKIE_PREFIX__ . 'use-build';
+		$isQuery = isset($_GET['use-dev']);
+		$cookieName = __SYM_COOKIE_PREFIX__ . 'use-dev';
 		$isCookie = isset($_COOKIE[$cookieName]);
+		$cookieTime = 2123798400;
 
 		if ($isQuery) {
+			if ($_GET['use-dev'] === 'no') {
+				$cookieTime = time() - 3600;
+				$isQuery = false;
+				$isCookie = false;
+			}
+
 			setcookie(
 				$cookieName,
 				'1',
-				2123798400,
+				$cookieTime,
 				__SYM_COOKIE_PATH__,
```

Comment:
We should make sure to only keep the .288dev.com part. maybe split on . and remove everything after the last two dots ?